docs/_includes/backers.md

```diff
@@ -1,7 +1,11 @@
 ## Backers
 
-Find Mocha helpful?  Become a [backer](https://opencollective.com/mochajs#support) and support Mocha with a monthly donation.
+Find Mocha helpful? Become a [backer](https://opencollective.com/mochajs#support) and support Mocha with a monthly donation.
 
-<!-- markdownlint-disable MD034 -->
-{% for i in (0..29) %}[![](https://opencollective.com/mochajs/backer/{{ i }}/avatar.jpg)](https://opencollective.com/mochajs/backer/{{ i }}/website){: target="_blank" rel="noopener"}{% endfor %}
-{: .image-list id="_backers" }
+{% comment %}
+Do not remove whitespace below!
+{% endcomment %}
+
+<ul class="image-list faded-images" id="backers">
+{% for supporter in supporters.backers %}<li>{% if supporter.website %}<a href="{{ supporter.website }}" target="_blank" rel="noopener" title="{{ supporter.name }}">{% endif %}<img src="{{ supporter.avatar }}" alt="{{ supporter.name }}" />{% if supporter.website %}</a>{% endif %}</li>{% endfor %}
```

Comment:
If an image fails to load, we'll see a broken image icon and the alt text, which messes up the layout. Lets make the image invisible to screen readers `alt=""` and put an `aria-label` on the anchor instead

```suggestion
{% for supporter in supporters.backers %}<li>{% if supporter.website %}<a href="{{ supporter.website }}" target="_blank" rel="noopener" title="{{ supporter.name }}" aria-label="{{ supporter.name }}">{% endif %}<img src="{{ supporter.avatar }}" alt="" />{% if supporter.website %}</a>{% endif %}</li>{% endfor %}
```